tox.ini

```diff
@@ -22,9 +22,7 @@ commands = pytest {posargs:tests/unit}
 [testenv:docs]
 description = Build documentation
 deps = -r{toxinidir}/docs/doc-requirements.txt
-commands = 
-  sphinx-apidoc -o docs/source/api ansible_sdk
-  sphinx-build -T -E -n --keep-going {tty:--color} -j auto -d docs/build/doctrees -b html docs/source docs/build/html
+commands = sphinx-build -T -E -W -n --keep-going {tty:--color} -j auto -d {temp_dir}/doctrees -b html docs/source docs/build/html
```

Comment:
(nit) I'd still keep it on a separate line for the sake of nice diffs and fitting things on screen.

Also, the `-W` and `{temp_dir}` could use a separate PR. Adding them is not related to the immaterial ext.

Reply:
Done with https://github.com/ansible/ansible-sdk/pull/43